### PR TITLE
Feature/22 form serializer for mobile app

### PR DESCRIPTION
**Pull request by wayangalihpratama**
### TODO / DONE

---

- [x] Create core_mobile module
- [x] Create MobileFormDefinition serializer and view
- [x] Create device endpoint to get form definition by id
- [x] Update question definition to add sqlite source param
- [x] Update all urls re_path (add `^` before route name)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205473308327980

### CODE REVIEW

File: backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        result = data.json()
+        self.assertEqual(result, {
+            "id": 1694489640833,
+            "name": "New Form",
+            "description": "New Form Description",
+            "defaultLanguage": "en",
+            "languages": ["en"],
+            "version": 1,
+            "cascades": [
+                "/sqlite/the_node.sqlite"
```
Comment (dedenbangkit) [inline backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py]: how does the mobile app download this sqlite?

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py (hunk 1 above)
Comment (wayangalihpratama) [reply backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py]: Maybe, we can use WEBDOMAIN env from docker to provide the complete url ?

Comment (dedenbangkit) [reply backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py]: Thanks, @anjarakvo let me know when the WEBDOMAIN is added

Comment (anjarakvo) [reply backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py]: I've just added it to the server manifest. Once this code is deployed, the WEBDOMAIN environment variable should be accessible.

Comment (dedenbangkit) [reply backend/akvo/core_mobile/tests/test_mobile_form_endpoint.py]: Thanks @anjarakvo

**Pull request merged**